src/Http/Response.php

```diff
@@ -0,0 +1,2432 @@
+<?php
```

Comment:
I can try and redo the changes. I suspect the current state happened because I had to update several typehints inside the code.